torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -1924,7 +1924,7 @@ Tensor cholesky_jvp(const Tensor& dA, const Tensor& L, bool upper) {
   //               = sym(L^{-1}dL)
   // where sym(X) = X + X^H
   // A short computation gives that the inverse of sym is given by
-  // \pi(X) = X.tril() - 0.5*diag(X)
+  // \pi(X) = X.tril() - 0.5*diag(X) when X is lower triangular,
```

Comment:
This is not correct. `sym : Tril(n) -> Sym(n)` so `\pi : Sym(n) -> Tril(n)` and `X` here is symmetric.

Reply:
I've reverted the changes to this comment.